src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -26,8 +28,8 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    if (timer.get() == 0 && shooter.canShoot()) { // TODO: Add "and intake has a peice" 
-      // TODO Move intake
+    if (timer.get() == 0 && shooter.canShoot()) {
+      intake.shoot(1);
```

Comment:
I can't tell what this parameter is very easily. My IDE would probably tell me so it's not the end of the world having to look somewhere else to know it's.... speed.